molpipeline/mol2any/mol2morgan_fingerprint.py

```diff
@@ -151,12 +151,11 @@ def _explain_rdmol(self, mol_obj: RDKitMol) -> dict[int, list[tuple[int, int]]]:
         dict[int, list[tuple[int, int]]]
             Dictionary with bit position as key and list of tuples with atom index and radius as value.
         """
-        bit_info: dict[int, list[tuple[int, int]]] = {}
-        _ = AllChem.GetMorganFingerprintAsBitVect(
-            mol_obj,
-            self.radius,
-            useFeatures=self._use_features,
-            bitInfo=bit_info,
-            nBits=self._n_bits,
+        fp_generator = self._get_fp_generator()
+        additional_output = AllChem.AdditionalOutput()
+        additional_output.AllocateBitInfoMap()
+        _ = fp_generator.GetSparseFingerprint(
```

Comment:
@c-w-feldmann Do you think we should call the super.transform or another parent class function instead, to ensure the same arguments to .Get*Fingerprint as they are used in the super's transform?